internal/server/data/organization.go

```diff
@@ -33,12 +39,39 @@ func CreateOrganization(tx WriteTxn, org *models.Organization) error {
 	if org.Name == "" {
 		return fmt.Errorf("Organization.Name is required")
 	}
+
+	pubkey, seckey, err := ed25519.GenerateKey(rand.Reader)
+	if err != nil {
+		return err
+	}
+
+	sec := jose.JSONWebKey{Key: seckey, KeyID: "", Algorithm: string(jose.ED25519), Use: "sig"}
+
+	thumb, err := sec.Thumbprint(crypto.SHA256)
+	if err != nil {
+		return err
+	}
+
+	sec.KeyID = base64.URLEncoding.EncodeToString(thumb)
+
+	pub := jose.JSONWebKey{Key: pubkey, KeyID: sec.KeyID, Algorithm: string(jose.ED25519), Use: "sig"}
+
+	secs, err := sec.MarshalJSON()
+	if err != nil {
+		return err
+	}
+
+	pubs, err := pub.MarshalJSON()
+	if err != nil {
+		return err
+	}
+
+	org.PrivateJWK = models.EncryptedAtRest(secs)
+	org.PublicJWK = pubs
```

Comment:
Not a blocker, but I guess in general we would only set these values if the fields are nil, and we'd probably set these things from the caller. It's probably fine to keep it here for now.